ballerina/client_endpoint.bal

```diff
@@ -165,6 +165,26 @@ public isolated client class DataPlaneClient {
         'class: "io.ballerinax.cosmosdb.DataplaneClient"
     } external;
 
+    // Patches a document.
+    // 
+    // + databaseId - ID of the database to which the container belongs
+    // + containerId - ID of the container which contains the document
+    // + documentId - ID of the document
+    // + partitionKey - The specific value related to the partition key field of the container
+    // + operations - An array of PatchOperation records specifying the operations to perform
+    // + requestOptions - Optional request options which can include a filterPredicate for conditional patching
+    // + return - DocumentResponse if successful, or error if failed
+    @display {label: "Patch Document"}
+    remote isolated function patchDocument(@display {label: "Database ID"} string databaseId,
+                                          @display {label: "Container ID"} string containerId,
+                                          @display {label: "Document ID"} string documentId,
+                                          @display {label: "Partition Key"} string partitionKey,
+                                          @display {label: "Patch Operations"} PatchOperation[] operations,
+                                          @display {label: "Optional Header Parameters"} PatchRequestOptions? requestOptions = ()) 
+                                          returns cosmosdb:DocumentResponse|error = @java:Method {
+        'class: "io.ballerinax.cosmosdb.DataplaneClient"
+    } external;
```

Comment:
@Atharva1723 You need to implement the required java utilities in the `io.ballerinax.cosmosdb.DataplaneClient` class (inside the `native/` directory) in order for this to work. As a reference you can check the [java implementation for the `createDocument` operation](https://github.com/ballerina-platform/module-ballerinax-azure-cosmosdb/blob/96b231a98cacd99081068e56e93a6acdceb5eaad/native/src/main/java/io/ballerinax/cosmosdb/DataplaneClient.java#L85)